emulated-integration-tests/src/rococo_network.rs

```diff
@@ -0,0 +1,25 @@
+pub use crate::chains::{asset_hub_rococo, pop_network, rococo};
+use crate::chains::{asset_hub_rococo::AssetHubRococo, pop_network::PopNetwork, rococo::Rococo};
+
+// Cumulus
+use emulated_integration_tests_common::{
+	accounts::{ALICE, BOB},
+	xcm_emulator::{decl_test_networks, decl_test_sender_receiver_accounts_parameter_types},
+};
+
+decl_test_networks! {
+	pub struct RococoMockNet {
```

Comment:
Not much going on in this file, can the contents to be moved to the top of lib.rs to simplify?